README.md

```diff
@@ -79,7 +79,7 @@ One of the primary advantage of CMake based build system is integration with oth
 1. Clone latest version:
 
   ```
-  git clone https://github.com/neuronsimulator/nrn
+  git clone --recursive https://github.com/neuronsimulator/nrn
```

Comment:
I don't know how/why this got into this pull request. It must come from a previous merge into the pyiter branch. We do NOT want to clone recursively in general as cmake does that as needed.